net-vpn/openfortivpn/openfortivpn-1.23.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2025 Gentoo Authors
```

Comment:
systemd use seems automagic. We should always pass `--with-systemdsystemunitdir` using `systemd.eclass` but the `libsystemd` use will need fixing aside from that.

There also seems to be automagicish behaviour based on whether openresolv is installed?

Reply:
OK.